### ISSUE TITLE
Not working for anything beyond System namespace 

### ISSUE THREAD

**Issue by daniel-scatigno** [natanfudge/Auto-Using#24]
This is not working for anything else System namespace
I'm working with asp.net core 3.1 
It doesn't not import anything from Microsoft.AspNet namespace
I'm using latest C# Omnisharp 1.21.15 on Linux

Can I provide more info? How?

**Comment by natanfudge** [comment]
This is a known limitation. 
I originally developed [branch 0.7](https://github.com/natanfudge/Auto-Using/tree/0.7) that solves this issue by introducing a C# language server. However, this conflicted with my intention of the extension being lightweight. At that time this feature was introduced into Visual Studio and I have expected for it to be implemented for vscode (https://github.com/OmniSharp/omnisharp-roslyn/issues/1482), so overall I lacked motivation. I also do not use C# anymore and if I would I would use Jetbrains Rider and not VSCode. 

The bottom line: I do not plan on expanding this extension further. However, seeing that this feature is not being implemented by the Omnisharp team, I would aid anyone in expanding the available namespaces, via [branch 0.7](https://github.com/natanfudge/Auto-Using/tree/0.7) which had NuGet  namespaces implemented already, or starting from the current, live branch [(master)](https://github.com/natanfudge/Auto-Using).

**Comment by BrandonBoone** [comment]
FYI: this is now available in OmniSharp: https://github.com/OmniSharp/omnisharp-roslyn/pull/1896

Setting is: `omnisharp.enableImportCompletion`
